moped-etl/prefect/tasks/activity_log.py

```diff
@@ -16,16 +16,14 @@
 SHA_SALT = os.environ["SHA_SALT"]
 
 # The folder name of the event function and used in naming the AWS function
-function_name = "activity_log"
+FUNCTION_NAME = "activity_log"
```

Comment:
I was clobbering this variable in a function inadvertently. As I uppercased it, it felt like it may want to jump out into the ENVIRONMENT, TBD.

Reply:
That is a good catch!